app.js

```diff
@@ -487,18 +485,6 @@ d.run(() => {
 						logLevel: 0,
 					};
 
-					if (scope.config.ssl.enabled) {
```

Comment:
We should think of it thoroughly and decide for some certain foreseen future. If we see that SSL option will never be part of P2P layer soon, or if it will, it will be seperate implementation. Then we should move `ssl` configuration key to under `api`. Then it will be clear that this `ssl` is only for API, not for p2p layer.

Reply:
This is definitely something that we should do in the future but one of the major goals of this PR is to make the migration from 0.9 to 1.0 as smooth as possible from the DevOps/SysAdmin point of view. Maintaining backwards compatibility with the old config file helps in that regard.

Reply:
This PR is created against `1.1.0`. Does it really have to do something with 0.9.x release? 

And there is no point where DevOps/SysAdmin involved here. Core itself have the configuration migration script, you can modify it as the way you want. That script always run during upgrade.

Reply:
@nazarhussain I'm referring to application migration in a broader sense, not database migration. When I say 'DevOps/SysAdmin', I'm referring to both independent people who run a Lisk node operators and also to people who operate Lisk nodes on behalf of large companies (e.g. exchanges).

Reply:
@nazarhussain agree - SSL will not be the part of the P2P layer for now. The current place of SSL config doesn't really fit and should be moved to "api" section.

Reply:
@nazarhussain I misunderstood what you meant about config migration earlier. I didn't realise we had such a script.